lib/sentry.service.ts

```diff
@@ -46,40 +46,65 @@ export class SentryService extends Logger {
   log(message: string, context?: string) {
     message = `${this.app} ${message}`;
     try {
-      Sentry.captureMessage(message, Sentry.Severity.Log);
       super.log(message, context);
+      Sentry.addBreadcrumb({
+        message,
+        level: Sentry.Severity.Log,
+        data: {
+          context
+        }
+      })
     } catch (err) {  }
   }
 
   error(message: string, trace?: string, context?: string) {
     message = `${this.app} ${message}`;
     try {
-      Sentry.captureMessage(message, Sentry.Severity.Error);
       super.error(message, trace, context);
+      Sentry.captureMessage(message, Sentry.Severity.Error);
```

Comment:
Any reason to move this after?

Reply:
If for some reason `Sentry.captureMessage` throws an error `super.error` won't be called. This way we ensure that at least the base method is called always.

Reply:
why not 
```typescript

try {
   Sentry.captureMessage(message, Sentry.Severity.Error);
} catch (e) {
   super.error(message, trace, context);
}

```